BoxContentSDK/BoxContentSDKTests/BOXContentClientTestCase.m

```diff
@@ -7,7 +7,78 @@
 //
 
 #import "BOXContentClientTestCase.h"
+#import "BOXContentClient.h"
+#import "BOXAPIAccessTokenDelegate.h"
+#import "BOXContentClient+Folder.h"
+#import "BOXContentClient_Private.h"
+#import "BOXFolderRequest.h"
+#import "BOXRequest_Private.h"
 
```

Comment:
Can you add a test to enforce what the expected behavior is if fetchAccessToken returns a nil access token?